src/server/KbnServer.js

```diff
@@ -83,4 +83,18 @@ module.exports = class KbnServer {
   async close() {
     await fromNode(cb => this.server.stop(cb));
   }
+
+  async inject(opts) {
+    if (!this.server) await this.ready();
+
+    return await fromNode(cb => {
+      try {
```

Comment:
I'm probably missing something... but is the `await` necessary here before returning? Since the value from `fromNode` isn't being used for anything else in the function, could it just return the promise?